django_rq/queues.py

```diff
@@ -72,21 +72,29 @@ def get_redis_connection(config):
         else:
             # We're using django-redis-cache
             return cache._client
-        
+
```

Comment:
We can make this a little bit cleaner by doing this:
```python
redis_class = redis.StrictRedis if use_strict_redis else redis.Redis

if 'UNIX_SOCKET_PATH' in config:
    return redis_class(unix_socket_path=config['UNIX_SOCKET_PATH'], db=config['DB'])

return redis_class(host=config['HOST'],
                   port=config['PORT'], db=config['DB'],
                   password=config.get('PASSWORD', None))
```
Mind making these changes? I'll merge this in right after